po/zh_CN.po

```diff
@@ -241,17 +255,21 @@ msgid ""
 "table or data.frame since they are lists too). To delete multiple columns "
 "use NULL instead. To add multiple empty list columns, use list(list())."
 msgstr ""
+"给定的 %d 列将被赋值为空列表（可能为空数据、"
+"表（table）或数据框（data.frame），因后两者也是列表的一种）。删除多个列时"
+"请使用空值（NULL）。添加多个空列表列（list columns）时，请使用 list(list())。"
 
 #: assign.c:395
 #, c-format
 msgid "Recycling single RHS list item across %d columns. Please unwrap RHS.\n"
-msgstr ""
+msgstr "右手侧（RHS）列表项循环应用于 %d 列。请拆开右手侧。\n"
 
 #: assign.c:397
 #, c-format
 msgid ""
 "Supplied %d columns to be assigned %d items. Please see NEWS for v1.12.2."
 msgstr ""
+"指定 %d 列赋值给 %d 项。请阅读 v1.12.2 的更新信息（NEWS）。"
```

Comment:
I think the assignment direction here need to be reversed.

提供的 %d 列将被赋值为  %d 项。请阅读 v1.12.2 的更新信息（NEWS）。

Reply:
Have added an FAQ about the possibility to reverse the order of the `%d` if you think it would be more natural; this is especially easy for C messages which are already in "printf"-style format : 

https://github.com/wch/r-source/blob/trunk/src/library/base/po/zh_CN.po#L1259-L1262

TL;DR is to use `%1$d` to refer to the `%d columns` input and `%2$d` to refer to the `%d items` input

Reply:
@MichaelChirico, being able to change the order of `%d` is great!

This error message comes from the case when `j` Does not have the same length as `values` in `set`. `提供的 %d 列将被赋值为 %d 项 ` sounds like the assignment is to be performed which it won’t.

@dracodoc , do you think `试图将 %2$d 项赋值给 %1$d 列` is an acceptable one? I add `试图` here to indicate this is a failure.

Reply:
In the beginning I though this might be a warning but still proceeded. If the NEWS reference is about No.1 in [NEWS](https://github.com/Rdatatable/data.table/blob/master/NEWS.md#new-features-2), then I agree 试图 is a better translation as the attempt failed.

Reply:
Yes, this refers to the NEWS you referenced. It changed from warning to error in [this](https://github.com/Rdatatable/data.table/pull/3310) pull request.

Reply:
You can trigger this error like so:

```
iris = as.data.table(iris)
set(iris, , c('x', 'y'), list(1, 2, 3))
```
```
Error in set(iris, , c("x", "y"), list(1, 2, 3)) : 
  Supplied 2 columns to be assigned 3 items. Please see NEWS for v1.12.2.
```